### PR TITLE
Add provider EliteTracker

### PR DESCRIPTION
**Pull request by adaur**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

Work based on https://github.com/SickRage/SickRage/pull/4013

### CODE REVIEW

**Comment by adaur** [comment]
There is. I started working on it but I'm facing a problem. The pubdate provided is like:

04.10.2017 01:36
DD.MM.YYYY HH:MM

When I send this to parse_pubdate the result is
2017-04-10 01:36:00+00:00
instead of
2017-10-04 01:36:00+00:00

I'm not sure how to reverse the order of day and month, some help would be appreciated. Thanks!

**Comment by labrys** [comment]
Adaur, the reason for the CI build failure is the following lines:
```
/home/travis/build/pymedusa/Medusa/medusa/providers/torrent/html/elitetracker.py:137:5: E125 continuation line with same indent as next logical line
/home/travis/build/pymedusa/Medusa/medusa/providers/torrent/html/elitetracker.py:153:40: E127 continuation line over-indented for visual indent
/home/travis/build/pymedusa/Medusa/medusa/providers/torrent/html/elitetracker.py:172:36: E127 continuation line over-indented for visual indent
/home/travis/build/pymedusa/Medusa/medusa/providers/torrent/html/elitetracker.py:175:29: E126 continuation line over-indented for hanging indent
/home/travis/build/pymedusa/Medusa/medusa/providers/torrent/html/elitetracker.py:181:28: E126 continuation line over-indented for hanging indent
/home/travis/build/pymedusa/Medusa/medusa/providers/torrent/html/elitetracker.py:188:1: D102 Missing docstring in public method
```
So mostly just indentation and a missing docstring.

**Comment by labrys** [comment]
As for the pubdate, the current `parse_pubdate` won't do that.  If it's always that format and never things like "5 seconds ago" or "now", then just use datetime directly.
```python
pubdate = datetime.strptime('04.10.2017 01:36', '%d.%m.%Y %H:%M')
```
If it's a mixture you could override `self.parse_pubdate` as follows:
```python
@staticmethod
def parse_pubdate(pubdate, human_time=False, timezone=None):
    try:
        return datetime.strptime(pubdate, '%d.%m.%Y %H:%M')
    except ValueError:
        return super(MyProvider, self).parse_pudate(pubdate)
```

**Comment by adaur** [comment]
Thank you very much @labrys I've used datetime directly. It should be good enough to be merged now, I'm not sure why Codacy fails, there is no blatant error for me.

File: medusa/providers/torrent/html/elitetracker.py [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+                        if mode != 'RSS':
+                            log.debug("Discarding torrent because it doesn't meet the"
+                                      " minimum seeders: {0}. Seeders: {1}",
+                                      title, seeders)
+                        continue
+
+                    torrent_size = torrent('td')[labels.index('Taille')].get_text(strip=True)
+                    size = convert_size(torrent_size) or -1
+
+                    pubdate_raw = torrent('td')[labels.index('Nom')].find_all('div')[-1].get_text().strip()
```
Comment (medariox) [inline medusa/providers/torrent/html/elitetracker.py]: Why not just ```.get_text(strip=True```)?

File: medusa/providers/torrent/html/elitetracker.py [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+            'submit': 'LOGIN',
+            'returnto': '/browse.php'
+        }
+
+        response = self.session.post(self.urls['login'], data=login_params)
+        if not response:
+            log.warning('Unable to connect to provider')
+            return False
+
+        if 'erreur' in response:
```
Comment (medariox) [inline medusa/providers/torrent/html/elitetracker.py]: Are you sure that you want to check for ```'erreur'``` in ```response``` and not in ```response.text```?

**Comment by adaur** [comment]
Should be good now, thanks for the review!

**Pull request merged**